digitalpublicgoods/santeims.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "SanteIMS",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/santedb/santedb-server/blob/master/SanteDB.licenseheader"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/santedb/dev-doc"
```

Comment:
This looks like a GitBook repo, it would be good to include the public GitBook URL for a more user-friendly read.

Reply:
Resolved in commit 931108